conda_smithy/lint_recipe.py

```diff
@@ -149,6 +149,14 @@ def lintify(meta, recipe_dir=None):
     except RuntimeError as e:
         lints.append(str(e))
 
+    # 13: There should be no comments
```

Comment:
Ok is there a good way to limit that down?
I have gotten a fair amount of boilerplate requests from reviewers to remove comments from my recipes.